packages/@uppy/locales/src/fi_FI.js

```diff
@@ -78,26 +78,26 @@ fi_FI.strings = {
   resume: 'Jatka',
   resumeUpload: 'Jatka lähetystä',
   retry: 'Yritä uudelleen',
-  retryUpload: 'Yritä uudelleen lähetystä',
+  retryUpload: 'Yritä lähetystä uudelleen',
   saveChanges: 'Tallenna muutokset',
   selectAllFilesFromFolderNamed: 'Valitse kaikki tiedostot kansiosta %{name}',
   selectFileNamed: 'Valitse tiedosto %{name}',
   selectX: {
-    '0': 'Valitse %{smart_count}',
-    '1': 'Valitse %{smart_count}',
-    '2': 'Valitse %{smart_count}'
+    '0': 'Valitse %{smart_count} tiedosto',
+    '1': 'Valitse %{smart_count} tiedostoa',
+    '2': 'Valitse %{smart_count} tiedostoa'
```

Comment:
I think this is wrong, because we’ve changed the string to specifically say “select X” instead of “select X files”, because there could also be folders, and we don’t want to make the button extra long.

Reply:
That's true. Those lines should be revised.  'tiedosto/tiedostoa' words are not needed.

Reply:
I’ll update later, thank you!